BitwardenShared/Core/Platform/Services/AuthenticatorSyncService.swift

```diff
@@ -160,16 +160,17 @@ actor DefaultAuthenticatorSyncService: NSObject, AuthenticatorSyncService {
         let decryptedCiphers = try await totpCiphers.asyncMap { cipher in
             try await self.clientService.vault(for: userId).ciphers().decrypt(cipher: cipher)
         }
-        let account = try await stateService.getActiveAccount()
-        let username = account.profile.email
+        let account = try? await stateService.getAccount(userId: userId)
 
         return decryptedCiphers.map { cipher in
             AuthenticatorBridgeItemDataView(
+                bitwardenAccountDomain: account?.settings.environmentUrls?.webVaultHost,
```

Comment:
Just want to double check with @fedemkr and @matt-livefront: This is the correct way to get the account domain, right? We found this being used in the profile selector, which is a similar use case for this.